src/main/resources/templates/snowflake/snowflake-schema-evolution/Makefile.ssp

```diff
@@ -34,15 +38,17 @@ evolve-schema:
     #elseif (tableDiff.allChangesAreCompatible(typeMapping))
 	$(MAKE) alter-snowpipe
 	$(MAKE) copy-into
+	mv $(currTableStateFile) $(previousStateDirectory)
 
     #elseif (!tableDiff.allChangesAreCompatible(typeMapping))
         <%
-            templateContext.addError(s"Incompatible change in table ${destination.stagingDatabase.name}.${destination.stagingDatabase.schema}.${tableDiff.destinationName} which will be ignored. Template: snowflake-schema-evolution/Makefile.ssp")
+            templateContext.addError(s"Incompatible change in table ${destination.stagingDatabase.name}.${destination.stagingDatabase.schema}.${tableDiff.destinationName}. User should analyse the changes and manually move ${tableDiff.destinationName}.yml from state-directory to previous-state-directory. Template: snowflake-schema-evolution/Makefile.ssp")
```

Comment:
> User should analyse the changes ...

User should resolve this incompatible change manually as per the documentation ....